common/types.go

```diff
@@ -781,3 +782,254 @@ func (b *BuilderSubmitBlockRequest) Withdrawals() []*consensuscapella.Withdrawal
 	}
 	return nil
 }
+
+// SubmitBlockRequest is the v2 request from the builder to submit a block.
+type SubmitBlockRequest struct {
```

Comment:
why not use the type from https://github.com/attestantio/go-builder-client/pull/8 ?